RELEASE.md

```diff
@@ -168,29 +168,73 @@ point but for now the process is manual.
 1. Open a Powershell terminal in the Windows virtual machine and execute:
 
    ```
-   $ cd c:\users\vagrant\signalfx-agent
-   $ scripts/windows/make.ps1 bundle -AGENT_VERSION "<X.Y.Z>"
+   PS> cd c:\users\vagrant\signalfx-agent
+   PS> .\scripts\windows\make.ps1 bundle -AGENT_VERSION "<X.Y.Z>"
    ```
 
    Where `<X.Y.Z>` is the release version.
 
-1. If the build is successful, verify that
-   `c:\users\vagrant\signalfx-agent\build\SignalFxAgent-X.Y.Z-win64.zip` exists.
+1. If the build is successful, verify that `.\build\SignalFxAgent-X.Y.Z-win64.zip` exists.
 
-1. If everything looks good, run the release script from your local workstation (must be
-   on the Splunk network):
+1. Run the signing script from your local workstation to sign the agent executable in
+   the bundle (must be on the Splunk network):
 
    ```
-   $ scripts/release --stage <STAGE> --push --new-version <X.Y.Z> --component windows --staging-token <repo.splunk.com token> --chaperone-token <chaperone token>
+   $ scripts/signing/sign_win_agent.py --staging-token <repo.splunk.com token> --chaperone-token <chaperone token> build/SignalFxAgent-X.Y.Z-win64.zip
    ```
 
-   Where `<STAGE>` is `test`, `beta`, or `release`,`<X.Y.Z>` is the same version from
-   step 1, and `<repo.splunk.com token>` and `<chaperone token>` are the API tokens for
-   the `srv-signalfx-agent` service account.
+   The signed bundle will be saved to `build/signed/SignalFxAgent-X.Y.Z-win64.zip`.
+
+1. Build the msi with the signed bundle in the Windows virtual machine:
```

Comment:
Ambiguous. Is the signed bundle in the Windows virtual machine? Should I build the msi with the signed bundle in a Windows virtual machine?